src/modules/tabs/__tests__/tabs.test.tsx

```diff
@@ -8,10 +8,11 @@ import ThemeProvider from '../../theming/theme-provider';
 import defaultTheme from '../../theming/default-theme';
 
 describe('Tabs', () => {
-  let tabItems: any, onChangeMock: any;
+  let tabItems: any, onChangeMock: any, onLinkPress: any;
```

Comment:
I don't mind adding the type annotations in unit tests. But if we are doing this, then let us make sure that we annotate them properly. I think using "any" is the same as not adding the annotations in the first place.

Reply:
Okay Ajay, will add specific types to each these props.